server/client/detectServer.ts

```diff
@@ -1,5 +1,7 @@
+import { tryGetEnv } from "../../util/mod.ts"
```

Comment:
Importing this from `util/mod.ts` will bring into a web-bundle `util/compression.ts`.
Then, `util/compression.ts` will bring `import { Buffer } from "../deps/std/streams.ts"`

I'm not sure if `compression.ts` should be excluded from `util/mod.ts`, or, if this import should be re-written to `import { tryGetEnv } from "../../util/tryGetEnv.ts"`